docs/artifactory/replicationtemplate/help.go

```diff
@@ -0,0 +1,8 @@
+package replicationtemplate
+
+const Description = "Create a JSON template for repository creation or update."
+
+var Usage = []string{`jfrog rt rtc <template path>`}
```

Comment:
Isn't "jfrog rt rtc" already taken for repo template?

Reply:
No, for repo template  that is `rpt`.